apps/docs/pages/guides/auth/managing-user-data.mdx

```diff
@@ -28,6 +28,12 @@ create table public.profiles (
 alter table public.profiles enable row level security;
 ```
 
+<Admonition type="note">
+
+Make sure you specify the `on delete cascade` clause whenever referencing `auth.users`. Not including it may cause problems when deleting users.
```

Comment:
```suggestion
Make sure to specify the `on delete cascade` clause when referencing `auth.users`. Omitting it may cause problems when deleting users.
```